README.md

```diff
@@ -110,8 +110,11 @@ Use the `addBodyPart` method to add a multipart part to the request.
 This part can be of type:
 * `ByteArrayPart`
 * `FilePart`
+* `InputStreamPart`
 * `StringPart`
 
+**NOTE**: `InputStreamPart` does not support zero-copy transfers. Explicitly disable zero-copy using `config.setDisableZeroCopy(true)` when one of the body parts is an `InputStreamPart`.
```

Comment:
Please revert

Reply:
Thanks for clearing that up.
I've reverted the changes and added a check to BodyChunkedInput if contentLength  0